### DISCUSSION TITLE
problem with running WeBots on windows - not able to initializing qt plug in

### DISCUSSION THREAD

**Discussion by amirhnourian** [cyberbotics/webots#5767] [General]
Hi everyone
today I installed webots on windows 11 using the source .exe file provided on the website but when I try to run the program I see this error message. 
Any opinion on what is the problem?
![Screenshot 2023-01-21 201317](https://user-images.githubusercontent.com/98447576/213877391-5ebe4f8e-237f-4664-afc5-6c702409e330.jpg)

**Comment by omichel** [comment]
Strange... Did you installed version [R2023a](https://github.com/cyberbotics/webots/releases/tag/R2023a) or a [nightly build](https://github.com/cyberbotics/webots/releases)?

**Reply by amirhnourian** [reply]
Dear omichel 

Thank you for prompt reply ;)

I have installed R2023a.

**Reply by omichel** [reply]
May I ask you to try also with a night build?

**Reply by amirhnourian** [reply]
I appreciate your follow-up. 

I tried all three nightly versions, but unfortunately, I received the same message. In the meantime, I have installed WeBots on my Ubuntu PC and it works like a charm. 

I could not solve the issue by searching on the internet too, maybe the issue is related to my own system!

Thank you again for your help and support

**Reply by omichel** [reply]
I would say it is indeed related with you local Windows machine as Webots works very well on Windows for many users.

**Reply by mwlock** [reply]
I actually got a similar error when trying to run R2023a on WSL2. I updated my WSL and then the error disappeared (there are other problems now though). Is it a coincidence that the messages are the same?

**Reply by omichel** [reply]
I don't know...

**Reply by amirhnourian** [reply]
> I actually got a similar error when trying to run R2023a on WSL2. I updated my WSL and then the error disappeared (there are other problems now though). Is it a coincidence that the messages are the same?

I think they may be unrelated, I tried to install docker a week ago so I installed WSL2 on my system. I have installed webots again, however I have the same error.
